### PR TITLE
Make the build manifest work on tags and other non-branch heads as well

### PR DESCRIPTION
**Pull request by bdunne**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,8 +3,8 @@ FROM registry.access.redhat.com/ubi8/ubi:8.3 AS manifest
 COPY .git /tmp/.git
 
 RUN cd /tmp && \
-    head=$(cat .git/HEAD |cut -d " " -f 2) && \
-    sha=$(cat .git/$head) && \
+    sha=$(cat .git/HEAD |cut -d " " -f 2) && \
+    if [[ $(cat .git/HEAD) == ref* ]];then sha=$(cat .git/$sha); fi && \
```
Comment (Fryguy) [inline Dockerfile]: Super-minor

```suggestion
    sha=$(cat .git/HEAD | cut -d " " -f 2) && \
    if [[ $(cat .git/HEAD) == ref* ]]; then sha=$(cat .git/$sha); fi && \
```

Comment (Fryguy) [reply Dockerfile]: Also wondering if we should enquote things for safety

```suggestion
    sha=$(cat .git/HEAD |cut -d " " -f 2) && \
    if [[ "$(cat .git/HEAD)" == "ref:"* ]];then sha=$(cat .git/$sha); fi && \
```

Comment (Fryguy) [reply Dockerfile]: Oh ignore the [[]] to [] change...didn't realize that * required the [[]]  (EDITed above)

**Comment by Fryguy** [comment]
Saw briefly you had `FROM 0` in there - did you want to try `FROM scratch` ?

**Review by Fryguy — APPROVED** [review]

**Comment by bdunne** [comment]
`scratch` doesn't even have `/bin/sh`, so that's a non-starter.  I switched to `ubi-minimal:latest` to be consistent across images.

**Pull request merged**